### PR TITLE
Feature/buttons

### PR DESCRIPTION
**Pull request by kamskr**
## Description

Add Button component with 2 variants

Screenshot 2021-07-15 at 01 36 52

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/lou_bank_ui/lib/src/lou_bank_ui.dart [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export 'colors/lou_colors.dart';
 export 'theme/lou_theme.dart';
 export 'typography/lou_typography.dart';
+export 'widgets/widgets.dart';
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/lou_bank_ui.dart:4]: wrong exports  'typography/typography.dart';

File: packages/lou_bank_ui/lib/src/widgets/lou_button.dart [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+
+import 'package:lou_bank_ui/lou_bank_ui.dart';
+
+enum LouButtonVariant {
+  darkGrey,
+  yellow,
+}
+
+class LouButton extends StatelessWidget {
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/widgets/lou_button.dart:5]: I would fo 2 widgets from it 

LouButton 
LouGradfientButton for clarity

File: packages/lou_bank_ui/lib/src/widgets/lou_button.dart [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                  colors: [
+                    LouColors.gradientYellowStart,
+                    LouColors.gradientYellowEnd,
+                  ],
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                ),
+                borderRadius: BorderRadius.circular(38),
+              ),
+              child: Container(
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/widgets/lou_button.dart]: We don't use containers cuz they have a lot of shit 

Padding -> Align -> SizedBox would be much better. 

If you read container file you will see that it is a widget that has multiple children. Use it only when you use most of them.

Research: ColoredBox, SizedBox, Align, Padding, Center (Align: Aligment.Center) and try to use the simplest widgets

File: packages/lou_bank_ui/lib/src/widgets/lou_button.dart [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                padding: const EdgeInsets.symmetric(vertical: 13),
+                margin: const EdgeInsets.all(0),
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/widgets/lou_button.dart]: Yyyy yeah no. If you need to do something like that then do EdgeInsets.zero

File: packages/lou_bank_ui/lib/src/widgets/lou_button.dart [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                width: double.infinity,
+                alignment: Alignment.center,
+                child: Text(text),
+              ),
+            ),
+          )
+        : ElevatedButton(
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/widgets/lou_button.dart]: A lot of repetition in this file Style could be taken out to thew method and the colors could be taken as params

Comment (kamskr) [reply packages/lou_bank_ui/lib/src/widgets/lou_button.dart]: Is it better now?

File: packages/lou_bank_ui/lib/src/widgets/lou_button.dart [hunk 6]
```diff
@@ -2,77 +2,84 @@ import 'package:flutter/material.dart';
...
+            colors: [
+              LouColors.gradientYellowStart,
+              LouColors.gradientYellowEnd,
+            ],
+            begin: Alignment.topCenter,
+            end: Alignment.bottomCenter,
+          ),
+          borderRadius: BorderRadius.circular(38),
+        ),
+        child: Container(
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/widgets/lou_button.dart]: Still using this shitty container

File: packages/lou_bank_ui/lib/src/widgets/lou_button.dart [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+    Key? key,
+    required this.text,
+  }) : super(key: key);
+
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 13),
```
Comment (jdebecka) [inline packages/lou_bank_ui/lib/src/widgets/lou_button.dart:104]: Material guildlines allow grid by 4

**Review by jdebecka — APPROVED** [review]

**Pull request merged**